src/sass/chrome.scss

```diff
@@ -189,3 +189,7 @@ section.ins-c-app-switcher--loading {
         text-align: inherit;
     }
 }
+
+.ins-chrome-content {
+    min-height: 100vh;
```

Comment:
I don't really like `vh` and `vw` units as they can sometime leads to unwanted scrollbars. Instead I'd set `height: 100%` on the `#page`

Reply:
I would not take this PR to reflect the finished thing. I am not even sure we will use the pf Page component because it does not work very well. I've mainly created this to make the demo look more like chrome instead of having just a blank page with 3 links.